configs/nas/greedynas/greedynas_subnet_mobilenet_8xb96_in1k.py

```diff
@@ -0,0 +1,170 @@
+_base_ = [
+    '../../_base_/datasets/mmcls/imagenet_bs128_colorjittor_aa_greedynas.py',
+    '../../_base_/schedules/mmcls/imagenet_bs1024_aa_greedynas.py',
+    '../../_base_/mmcls_runtime.py'
+]
+
+data = dict(samples_per_gpu=96)
+optimizer = dict(lr=0.048)
+lr_config = dict(
+    step=int(1666 * 2.4), warmup_iters=1666 * 3, warmup_ratio=1e-6 / 0.048)
```

Comment:
Does `1666` here indicate the number of iterations per epoch?

But according to the instruction [here](https://image-net.org/download.php), the number of total training images is `1281167`, and if batch size is `96 * 8 = 768`, the iterations of one epoch might be `ceil(1281167 / 768) = 1669` since `drop_last` is set to `True`

Reply:
Yes, but training dataset is splited for trianing and evaluating,  (1281167 - 1024) // 768 = 1666

Reply:
got it.